scripts/manage_case

```diff
@@ -1,375 +1,172 @@
+            break
+
+    # If name is not a valid argument - exit with error
+    expect(match_found != "unset", 
+           "Invalid input argument %s, valid input arguments are %s" % (name, components))
+
+    # Determine the config_file for the target component 
+    config_file = files.get_value("COMPSETS_SPEC_FILE", attribute={"component":name}, resolved=True)
+    expect((config_file),
+           "Cannot find any config_component.xml file for %s" %name)
+
+    # Check that file exists on disk
+    expect(os.path.isfile(config_file),
+           "Cannot find config_file %s on disk" %config_file)
+
+    # Now parse the compsets file and write out the compset alias and longname as well as the help text
+    # determine component xml content 
+    parsed_xml = Compsets(config_file)
+    parsed_xml.print_values()
+
+###############################################################################
+def query_component(name):
+###############################################################################
+
+    # Determine the valid component classes (e.g. atm) for the driver/cpl
+    # These are then stored in comps_array
+    files = Files()
+    infile = files.get_value("CONFIG_DRV_FILE")
+    comps_object = Component(infile)
+    comps_array = comps_object.get_value("component")
+    
+    # Loop through the elements for each component class (in config_files.xml) 
+    # and see if there is a match for the the target component in the component attribute 
+    match_found = False
+    valid_components = list()
+    for comp in comps_array:
+        string = "CONFIG_%s_FILE"%comp 
+
+        # determine all components in string
+        components = files.get_components(string, resolved=True)
+        for item in components:
+            valid_components.append(item)
+
+        # determine if config_file is on disk
+        config_file = files.get_value(string, attribute={"component":name}, resolved=True)
+        if config_file is not None:
+            match_found = True
+            expect(os.path.isfile(config_file),
+                   "Cannot find config_file %s on disk" %config_file)
+            break
+
+    # If name is not a valid argument - exit with error
+    expect((match_found), 
+           "Invalid input argument %s, valid input arguments are %s" % (name, valid_components))
+
+    # Check that file exists on disk, if not exit with error
+    expect((config_file),
+           "Cannot find any config_component.xml file for %s" %name)
+
+    # determine component xml content 
+    parsed_xml = Component(config_file)
+    parsed_xml.print_values()
+
+###############################################################################
+def parse_command_line(args):
+###############################################################################
+
+    cime_model = CIME.utils.get_model()
+
+    parser = argparse.ArgumentParser()
+
+    CIME.utils.setup_standard_logging_options(parser)
+
+    parser.add_argument("--query_compsets_setby", 
```

Comment:
It is not POSIX compliant to have --long_options_like_this. You want --long-options-like-this.